docs/source/markdown/podman-image-untag.1.md

```diff
@@ -0,0 +1,35 @@
+% podman-untag(1)
```

Comment:
Why is the necessary?

cat docs/source/markdown/links/podman-image-untag.1 
.so man1/podman-untag.1

The man pages are already linked together?

Reply:
Thanks! Was a brain fart on my side.